src/renderer/components/Modal/Profile.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <modal name="profile-modal" @before-open="beforeOpen">
+        <h1>Create a profile</h1>
+
+        <p>
+            <label for="column-name"> Profile Name</label>
+            <input type="text" for="column-name" v-model="profile_name"/>
```

Comment:
Please change this to profile-name instead of `column-name`

Reply:
Also change the model to use `profileName` instead of `profile_name` as noted below!